src/utils/line.ts

```diff
@@ -243,6 +248,19 @@ export const formatSecuredLineData = (
     [BigNumber.from(0), {}]
   );
 
+  // Create array of CreditEvent[]
+  const creditEvents: CreditEvent[] = eventFrags.map((event) => {
```

Comment:
this is what `formatCreditEvents` is for

Reply:
fixed.